desktop/apps/consignments2/components/thank_you/index.styl

```diff
@@ -0,0 +1,39 @@
+.consignments2-submission-thank-you
+  max-width 1125px
+  text-align center
+  margin 0 auto
+
+  &__submitted-work
+    width 100%
+    background-color gray-lightest-color
+    height 120px
+    display flex
+    align-items center
+
+  &__submitted-message
+    garamond-size('l-body')
+    margin-right 20px
+
+  &__submitted-image img
```

Comment:
I'd drop the `img` selector. You can put the BEM class directly on that tag and drop the containing `div`. But even if you needed the container, it would be best to give it a separate element name.